docs/source/algorithms/LoadILLSANS-v1.rst

```diff
@@ -9,13 +9,22 @@
 Description
 -----------
 
-This loads the nexus files produced by the SANS instruments D11, D22, D33 at the ILL.
-For D33, it supports both monochromatic and TOF modes.
+This loads the nexus files produced by the SANS instruments D11, D22, D33 and D16 at the ILL.
```

Comment:
That is covered by the mention later that it also supports newer version of those instruments, since D22B also exists and the list will start to be a bit long.